scraper.py

```diff
@@ -190,7 +198,7 @@ def scrape(output_file_name:str, no_of_articles:int, category_urls:dict, time_de
 
     with open(output_file_name, "w") as csv_file:
         headers = ["headline", "text", "category", "url"]
-        writer = csv.DictWriter(csv_file, delimiter=",", fieldnames = headers)
+        writer = csv.DictWriter(csv_file, delimiter="\t", fieldnames = headers)
```

Comment:
any reason for the `tsv` preference?

Reply:
The `text` column is lengthy and may contain one or more commas already. Using `tsv` made it easier to understand what's in the file at a glance.